packages/analytics/src/definitions.ts

```diff
@@ -71,6 +71,38 @@ export interface FirebaseAnalyticsPlugin {
    * @since 0.1.0
    */
   resetAnalyticsData(): Promise<void>;
+  /**
+   * Initiates on-device conversion measurement with an email address.
+   *
+   * Only available for Android and iOS.
+   *
+   * @since 6.0.0
```

Comment:
```suggestion
   * @since 7.2.0
```

The next version will be 7.2.0.